<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1220</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1200</width>
     <height>700</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1200</width>
     <height>700</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1200</width>
     <height>700</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frame{
	border-image:url(:/background/images/background/bg3.jpg)
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>350</width>
      <height>510</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QListWidget" name="listLocal">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>350</width>
        <height>510</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget{
font: 75 12pt &quot;Comic Sans MS&quot;;
background-color:transparent;

}
QListWidget::item{
color:rgb(120,120,120);
border:none;
margin-top:3px;

}
QListWidget::item:hover{
padding-left:4px;
margin-top:3px;
color:rgb(100,100,100);
background-color:rgba(255,255,255,0.2);

}
QListWidget::item:selected{
outline:none;
margin-top:3px;
padding-left:4px;
background-color:rgba(255,255,255,0.5);
color:rgb(0,0,0);
}
QListWidget::item:!active{
margin-top:3px;
border:none;
background-color:rgba(255,255,255,0);
color:rgb(120,120,120);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QListWidget" name="listLike">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>350</width>
        <height>510</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget{
font: 75 12pt &quot;Comic Sans MS&quot;;
background-color:transparent;

}
QListWidget::item{
color:rgb(120,120,120);
border:none;
margin-top:3px;

}
QListWidget::item:hover{
padding-left:4px;
margin-top:3px;
color:rgb(100,100,100);
background-color:rgba(255,255,255,0.2);

}
QListWidget::item:selected{
outline:none;
margin-top:3px;
padding-left:4px;
background-color:rgba(255,255,255,0.5);
color:rgb(0,0,0);
}
QListWidget::item:!active{
margin-top:3px;
border:none;
background-color:rgba(255,255,255,0);
color:rgb(120,120,120);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QListWidget" name="listPlay">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>350</width>
        <height>510</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget{
font: 75 12pt &quot;Comic Sans MS&quot;;
background-color:transparent;

}
QListWidget::item{
color:rgb(120,120,120);
border:none;
margin-top:3px;

}
QListWidget::item:hover{
padding-left:4px;
margin-top:3px;
color:rgb(100,100,100);
background-color:rgba(255,255,255,0.2);

}
QListWidget::item:selected{
outline:none;
margin-top:3px;
padding-left:4px;
background-color:rgba(255,255,255,0.5);
color:rgb(0,0,0);
}
QListWidget::item:!active{
margin-top:3px;
border:none;
background-color:rgba(255,255,255,0);
color:rgb(120,120,120);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QListWidget" name="listHistory">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>350</width>
        <height>510</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget{
font: 75 12pt &quot;Comic Sans MS&quot;;
background-color:transparent;

}
QListWidget::item{
color:rgb(120,120,120);
border:none;
margin-top:3px;

}
QListWidget::item:hover{
padding-left:4px;
margin-top:3px;
color:rgb(100,100,100);
background-color:rgba(255,255,255,0.2);

}
QListWidget::item:selected{
outline:none;
margin-top:3px;
padding-left:4px;
background-color:rgba(255,255,255,0.5);
color:rgb(0,0,0);
}
QListWidget::item:!active{
margin-top:3px;
border:none;
background-color:rgba(255,255,255,0);
color:rgb(120,120,120);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font: 75 pt &quot;Comic Sans MS&quot;;
color:rgba(255,255,255,0.5);
}</string>
    </property>
    <property name="text">
     <string>Later Music</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: rgba(255, 255, 255,150);
border-style:outset;

border-width:0px;

border-radius:10px;
border-color: rgba(0,0,0,100);
font:bold 14px;
color:rgb(180,180,180);
padding:6px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="searchBtn">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>20</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-image:url(:/icon/images/icon/sereach.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{


background-image:url(:/icon-hover/images/icon-hover/sereach-hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image:url(:/icon/images/icon/sereach.png);
background-repeat:no-repeat;
background-position:center center;

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="localListBtn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-image:url(:/icon/images/icon/music.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{


background-image:url(:/icon-hover/images/icon-hover/music-hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image:url(:/icon/images/icon/music.png);
background-repeat:no-repeat;
background-position:center center;

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="likeListBtn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-image:url(:/icon/images/icon/like.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{


background-image:url(:/icon-hover/images/icon-hover/like-hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image:url(:/icon/images/icon/like.png);
background-repeat:no-repeat;
background-position:center center;

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="listBtn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-image:url(:/icon/images/icon/list.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{


background-image:url(:/icon-hover/images/icon-hover/list-hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image:url(:/icon/images/icon/list.png);
background-repeat:no-repeat;
background-position:center center;

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="recentlyListBtn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-image:url(:/icon/images/icon/play-resently.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{


background-image:url(:/icon-hover/images/icon-hover/play-resently-hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image:url(:/icon/images/icon/play-resently.png);
background-repeat:no-repeat;
background-position:center center;

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="prevBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>650</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-image:url(:/icon/images/icon/pre.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{


background-image:url(:/icon-hover/images/icon-hover/pre-hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image:url(:/icon/images/icon/pre.png);
background-repeat:no-repeat;
background-position:center center;

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="playOrPauseBtn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>640</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="nextBtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>650</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-image:url(:/icon/images/icon/next.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{


background-image:url(:/icon-hover/images/icon-hover/next-hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image:url(:/icon/images/icon/next.png);
background-repeat:no-repeat;
background-position:center center;

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="mySlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>339</x>
      <y>650</y>
      <width>641</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::add-page:Horizontal{     
	background-color: rgba(255, 255, 255,100);
	height:4px;  
} 
QSlider::sub-page:Horizontal{
	background-color: rgb(255, 255, 255);
	height:4px;
}
QSlider::groove:Horizontal{ 
	background:transparent; 
	height:4px;
}  
QSlider::handle:Horizontal{ 
	height:16px; 
	width:16px; 
	border-image:url(:/icon/images/icon/进度条.png);
	margin: -6 0px; 
}
      
</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="welcomLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>300</y>
      <width>541</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 12pt &quot;Comic Sans MS&quot;;
color: rgb(75, 75, 75);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="quitBtn">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-image:url(:/icon/images/icon/quit.png);
	background-repeat:no-repeat;
	background-position:center center;
	border:none;
}
QPushButton:hover{
	background-color: rgb(255, 255, 255,100);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="addMusicBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-image:url(:/icon/images/icon/add.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{


background-image:url(:/icon-hover/images/icon-hover/add-hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image:url(:/icon/images/icon/add.png);
background-repeat:no-repeat;
background-position:center center;

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="markLikeBtn">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>650</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-repeat:no-repeat;
	background-position:center center;
	border:none;
}

QPushButton:hover{
	background-repeat:no-repeat;
	background-position:center center;
}
QPushButton:pressed{
	background-repeat:no-repeat;
	background-position:center center;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="playModeBtn">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>650</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-repeat:no-repeat;
	background-position:center center;
	border:none;
}

QPushButton:hover{
	background-repeat:no-repeat;
	background-position:center center;
}
QPushButton:pressed{
	background-repeat:no-repeat;
	background-position:center center;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="volumeBtn">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>650</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-image:url(:/icon/images/icon/music-voice.png);
	background-repeat:no-repeat;
	background-position:center center;
	border:none;
}

QPushButton:hover{
	background-image:url(:/icon-hover/images/icon-hover/music-voice-hover.png);
	background-repeat:no-repeat;
	background-position:center center;
}
QPushButton:pressed{
	background-image:url(:/icon/images/icon/music-voice.png);
	background-repeat:no-repeat;
	background-position:center center;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="timeProgress">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>610</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Comic Sans MS&quot;;
color: rgb(98, 98, 98)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="musicInfo">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>600</y>
      <width>461</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;宋体&quot;;
color:rgb(98, 98, 98)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>510</y>
      <width>35</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider{
	background-color: rgba(255,255,255,0.2);
	min-width:5px; 
	max-width:5px;
	border:15px solid rgba(255,255,255,0.2); 
}
QSlider::add-page:vertical{      
	background-color: rgb(250, 250, 0);
	width:4px;
}
QSlider::sub-page:vertical{
	background-color: rgba(255,255,255,0.5); 
	width:4px;
}
QSlider::groove:vertical{
	background:transparent;
	width:6px;
}
QSlider::handle:vertical{
	height: 13px;
	width:13px;
	margin: -0 -4px; 
	border-image:url(:/icon/images/icon/圆形.png);
}</string>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mySlider</class>
   <extends>QSlider</extends>
   <header>myslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
